### PR TITLE
Replace dev expressions

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

File: packages/react-router-dom/.size-snapshot.json [hunk 1]
```diff
@@ -1,26 +1,26 @@
 {
   "esm/react-router-dom.js": {
-    "bundled": 7968,
-    "minified": 4870,
-    "gzipped": 1613,
+    "bundled": 7978,
+    "minified": 4880,
+    "gzipped": 1618,
```
Comment (mjackson) [inline packages/react-router-dom/.size-snapshot.json]: Any idea why these numbers are bigger than before @TrySound? Does it matter? Or are we only concerned with the UMD build sizes?

Comment (TrySound) [reply packages/react-router-dom/.size-snapshot.json]: I don't know. Maybe because of workspaces.

Comment (timdorr) [reply packages/react-router-dom/.size-snapshot.json]: It's 10 bytes, so I'm guessing the addition of the word `tiny-` in the code :)

Comment (TrySound) [reply packages/react-router-dom/.size-snapshot.json]: Ah, yeah. Right.

**Pull request merged**